frontend/src/metabase/visualizations/lib/action.js

```diff
@@ -20,7 +19,6 @@ export function performAction(action, { dispatch, onChangeCardAndRun }) {
       open(url, {
         openInSameOrigin: location => {
           dispatch(push(location));
-          dispatch(setParameterValuesFromQueryParams(location.query));
         },
```

Comment:
Doesn't it conflict with #24762? (that PR added this line)
Not sure how its related to tabs

Reply:
Yes it does, but only when routing to questions. I did not notice this until looking at the failing tests and when working on another issue. Removing that line does fix the race condition of pushing a new dashboard + tab location when performing a click action. Somehow it is competing for setting the URL when this occurs. 

I am investigating alternative fixes because it solves one bug but causes another.